.github/in-solidarity.yml

```diff
@@ -11,4 +11,4 @@ ignore:
  - ".github/in-solidarity.yml"  # default
  - ".github/workflows/*.yml"
  - "**/*.yml"
- - "app/manifests/install-config-template.j2"
+ - "app/manifests/*.j2"
```

Comment:
@rnetser @myakove do I need to update this file before PR, as `Inclusive Language`  check is failing for `openshift_cli_installer/manifests/gcp-install-config-template.j2`

Reply:
please add to in-solidarity config